sdk/ml/azure-ai-ml/tests/test_configs/pipeline_jobs/serverless_compute/all_types/automl/pipeline_with_vm_size_no_default.yml

```diff
@@ -0,0 +1,47 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
```

Comment:
maybe name this from vm_size -> instance_type

Reply:
Renamed.